### PR TITLE
docs(dedibox): update faq

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Update Dedibox Link FAQ

### CODE REVIEW

**Review by ofranc — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

File: faq/dedibox-in-scaleway-console.mdx [hunk 1]
```diff
@@ -114,7 +114,7 @@ Yes, the API system is not impacted by account linking.
 
 
 ## What are the terms of the voucher offered for the migration?
-For the first customers following the linking process, a discount of 100€ is applied directly to their Scaleway Elements account. This discount is valid for six months after the link of your accounts. It is available on our Scaleway Elements cloud services ordered through the Scaleway console, excluding DDX and Elastic Metal products.
+For the first customers following the linking process, a discount of 100€ is applied directly to their Scaleway account. This discount is valid for six months after the link of your accounts. It is available on our Scaleway cloud services ordered through the Scaleway console, excluding Dedibox and Elastic Metal products.
```
Comment (farhakennar) [inline faq/dedibox-in-scaleway-console.mdx:117]: The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.

File: faq/dedibox-in-scaleway-console.mdx [hunk 2]
```diff
@@ -114,7 +114,7 @@ Yes, the API system is not impacted by account linking.
...
 
 ## How can I ask my remaining questions?
```
Comment (farhakennar) [inline faq/dedibox-in-scaleway-console.mdx:119]: Who can I ask if I have further questions?

File: faq/dedibox-in-scaleway-console.mdx [hunk 3]
```diff
@@ -95,17 +95,17 @@ In the future, you may only need to use your Scaleway login information to conne
 
 ## What impact will linking accounts have on my billing?
 
-You will continue to receive an invoice for each type of product (one for Dedibox and one for Scaleway Elements products).
-However, the default payment method defined in the Elements console will allow you to pay all your Scaleway invoices (Dedibox and Elements). It is important that you verify the default payment method defined in the Scaleway console after the linking process, to ensure the proper payment method will be used for your future invoices.<br />
+You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products).
+However, the default payment method defined in the Scaleway console will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is important that you verify the default payment method defined in the Scaleway console after the linking process, to ensure the proper payment method will be used for your future invoices.<br />
```
Comment (farhakennar) [inline faq/dedibox-in-scaleway-console.mdx:99]: However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.

**Review by farhakennar — APPROVED** [review]

File: faq/dedibox-in-scaleway-console.mdx (hunk 1 above)
Comment (bene2k1) [reply faq/dedibox-in-scaleway-console.mdx:117]:
```suggestion
The first customers to follow the linking process will be granted a discount of 100€. This discount is directly applied to the Scaleway account and is valid for six months following the linking process. It is available on Scaleway cloud services ordered through the Scaleway console and does not include Dedibox and Elastic Metal products.
```

File: faq/dedibox-in-scaleway-console.mdx (hunk 2 above)
Comment (bene2k1) [reply faq/dedibox-in-scaleway-console.mdx:119]:
```suggestion
## Who can I ask if I have further questions?
```

File: faq/dedibox-in-scaleway-console.mdx (hunk 3 above)
Comment (bene2k1) [reply faq/dedibox-in-scaleway-console.mdx:99]:
```suggestion
However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.<br />
```

File: faq/dedibox-in-scaleway-console.mdx [hunk 4]
```diff
@@ -1,22 +1,22 @@
 ---
 meta:
-  title: Dedibox to Elements
+  title: Dedibox in Scaleway console
   description: Discover how to link your Dedibox servers to your Scaleway account.
 content:
-  h1: Dedibox to Elements
+  h1: Dedibox in Scaleway console
```
Comment (ldecarvalho-doc) [inline faq/dedibox-in-scaleway-console.mdx:6]:
```suggestion
  h1: Dedibox in the Scaleway console
```

File: faq/support.mdx [hunk 5]
```diff
@@ -9,7 +9,7 @@ hero: assets/general.webp
 ---
 
 ## What is my current support plan?
-You can check your support plan on [the support plan page within the Scaleway console](https://console.scaleway.com/support/plans). The free Basic support plan is included for all Scaleway Elements customers automatically.
+You can check your support plan on [the support plan page within the Scaleway console](https://console.scaleway.com/support/plans). The free Basic support plan is included for all Scaleway customers automatically.
```
Comment (nedaehm) [inline faq/support.mdx:12]:
```suggestion
You can find your support plan on [the support plan page within the Scaleway console](https://console.scaleway.com/support/plans). The free Basic support plan is included for all Scaleway customers automatically.
```

File: compute/dedibox/how-to/link-dedibox-account.mdx [hunk 6]
```diff
@@ -43,7 +43,7 @@ You can now manage your current and future Dedibox servers from the Scaleway con
     </Message>
 6. Select the personal and the billing information you wish to use for your Scaleway account. You can either use the personal and the billing information of your selected Scaleway Organization or the information linked to your Dedibox account. Then tick the checkboxes to confirm that:
     * You have taken note of your login for the Scaleway console.
-    * Your Dedibox and Scaleway Elements account information will be updated with the information given on this page.
+    * Your Scaleway Dedibox and Scaleway Cloud account information will be updated with the information given on this page.
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply compute/dedibox/how-to/link-dedibox-account.mdx:46]: Yes

**Pull request merged**